### DISCUSSION TITLE
Storage Factory Quiz

### DISCUSSION THREAD

**Discussion by MutaiAllan** [Cyfrin/foundry-full-course-cu#2959] [Ideas]
I think the answers should be updated, the return type must be included only when it is include in the parent function, Any views?

TakeAway:
Return Types, Parameter Types, Visibility and Mutability cannot be overridden.( I stay corrected.)

![Screenshot_2024-12-02_02_47_02](https://github.com/user-attachments/assets/d674f8cd-e887-42c8-a3cd-5cf194fd6e5d)

**Comment by EngrPips** [comment]
The answer is technically correct. I don't think there is a need to be that explicit.
